src/ray/raylet/node_manager.cc

```diff
@@ -14,6 +14,8 @@
 
 namespace {
 
+const bool USE_NEW_SCHEDULER = true;
```

Comment:
Consider adding this to `ray_config_def.h` and referencing it as `initial_config.use_new_scheduler` in node manager.cc